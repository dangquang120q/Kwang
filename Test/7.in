54
42
1938417 7574585 3941108 2295360 5248251 1949658 5276030 287085 2307298 2922204 6500532 534739 207698 7190299 6956995 2723142 420446 4846327 3100660 183992 8329561 6393566 9963252 1820541 4828065 5225474 5690703 4070246 2479463 7160000 928366 7803097 1287845 2632995 7134066 9096447 9531936 4379827 3359810 2856545 2461504 8157862 
34
7190369 9006223 6595379 4997591 8971380 9244981 4926944 4980417 8161930 374383 5198227 1942198 2416612 2634573 7843392 7584054 9974901 2936696 1766453 9789785 2068213 5428487 1384032 2772099 5120326 4823485 4627031 2260962 8985309 8208231 2525246 3428444 4061451 682966 
49
7785229 3148706 811582 6070642 6151116 5498932 3848706 7592486 638830 9240479 9896249 3727151 1531651 7047913 2426482 5551662 8287329 9226326 1496936 3946594 7473060 7596219 2650053 3317935 8746869 9403090 4520584 8680516 7518168 3101584 9819705 3225663 5462522 1314004 8933705 9049894 9723273 3403245 3148786 9977872 3568268 8872419 3471294 8055786 8267670 1352838 2601685 6077942 7144826 
43
2477250 9333006 7032012 7021853 7440885 1595347 7329982 9077752 8886241 140568 7362585 8023888 7571581 1097635 4294274 7109634 5093282 8932878 4248809 7951367 4742553 9561182 8722249 1070561 5305209 9523515 2178832 5885966 6258160 3043111 2038635 7093940 4900370 5719539 6045544 5142986 3081315 4366181 6407850 446108 5026291 6401651 6754231 
49
1664667 1351174 1344667 1181570 8121597 9758483 703080 8763685 3837228 7384145 8105867 816806 6735559 8002773 3560802 6146200 3236933 4990960 2025314 7563663 1749087 3298465 5516443 4099935 7479597 2417243 8116134 2658150 3214725 5018019 9367442 4701868 283630 1722549 6264830 5767428 5895312 5470493 382645 8309506 6112862 4367007 1364381 565660 2810212 9298843 3464317 2795691 3790037 
70
4037428 9466216 2398281 9302321 478860 6967021 7824198 3600544 3729609 7891485 7654828 2786067 3872277 307182 3290141 8647917 3318628 9067133 1555911 8046900 3144676 457215 9467841 2383280 4123559 1350720 8564138 5724882 6359523 5826444 3526323 4543002 2400755 136167 5188755 6880508 5225419 4305735 9374072 439809 9414149 170664 1845663 5924634 6971503 7771443 8507064 5928727 2312367 9751741 808496 3697567 7196344 6203529 6763633 9572974 2853076 1888639 7393406 4396994 1336187 5593475 1795052 9237580 9068982 7368795 4736334 2572071 4855859 2933398 
16
4710551 8170714 8241533 2914384 4043427 5055288 8813120 2936708 4782489 773736 1891516 5071016 6254963 3470804 3113692 8391142 
90
1406876 1405778 8127347 3846686 7430770 4443587 2242138 6846609 7154521 4348122 828499 4859010 7627132 7416425 9579869 4190049 2290513 1752378 6128312 8566780 4887340 9063431 3661935 4850214 2321999 4064924 8577913 5780099 5326445 3877136 2431614 2686881 7952212 3415266 8926954 2805964 2923998 2128889 9702930 1253905 8576085 5493734 7082353 9885280 1745536 9916136 9520755 1822748 5289192 7243160 8513883 700116 375169 2774882 2906782 1221520 7119467 2891317 9997415 5113651 3948600 4291707 265187 5581267 2421456 3763725 6933291 8970452 1395808 1423913 1259738 6330515 6294071 5294862 9597897 8142143 2021844 6743754 2711579 6023752 4038028 5770307 7079860 6345922 8260232 1090835 4692624 1792618 8504947 8101482 
22
9289764 4322327 717789 2137665 8735141 8442354 3576038 2140163 5379704 3828474 4041949 3170685 1733787 3550943 9931986 2735995 6009959 6888849 4257253 7044362 8505513 3440837 
93
259476 7036275 7543403 9430618 6597369 9991588 1241458 8885253 7905910 2427134 3408204 7825147 7575280 6256757 8166914 1652652 5139522 1114925 9292613 5421292 4117743 3429499 8297246 1669598 9027096 7993290 1511735 6041531 6323480 3362350 5337017 1197383 2609089 6426175 4886735 1358586 2783588 2305160 7985342 2955517 5820030 9735479 2726941 4968307 2567666 2514869 224723 308138 6825935 5208617 5564219 3445866 1022200 9710810 8275786 9136009 3050626 7871916 291386 4726653 8839366 9847247 4433425 5188898 4903672 3938839 7957628 9514623 4519443 5223713 5215671 1730874 9242389 4749249 6908565 5005563 7001131 8227875 2694604 431647 2992595 9532809 6938852 1041203 5905908 9600428 7402016 426230 7163092 1732171 2449333 8112367 2570567 
73
7270803 2465281 4561277 2431780 5362335 6061654 4990666 1561593 5698876 8092406 9016594 2480112 1025856 8182943 4706149 487949 8384988 5130498 7955893 1162825 7560016 7576760 5563108 2378751 2584669 2311013 4721951 3363550 9250957 4825905 4018403 2481450 9573817 4761573 6937235 3919140 7267884 7744681 7015123 9192859 619770 7840243 8460684 5774289 6467725 5664226 2122025 588609 6098959 3723525 7721976 7306893 2939857 232006 9327220 5816310 5259567 7353143 5659661 6553737 2100647 5854391 7595211 9365228 4936475 6713428 4367086 9742423 910425 5093376 9547401 4897483 9045587 
33
2322299 5414607 2063947 2088247 359097 6623204 6104718 4267139 9042432 7124103 1946037 6847974 6648469 9512224 9548318 4871354 5108089 2801063 6092915 8267723 1025348 2030465 2076096 718646 6066143 1972434 2775836 911103 200608 2620674 2480285 5493730 602705 
77
5911035 9491293 8698646 3061242 4271244 9490621 7611107 2937497 2586308 6988093 4901234 3626301 3455717 9280138 7283191 1704778 9947076 3743402 1582260 653012 7893821 3444245 8346171 3855264 574335 2066508 9684719 772776 9742722 8875594 907982 5110322 4133425 1183158 5394120 641884 8974292 7949938 1461357 8078575 7827651 2573463 6860954 4361095 855384 8016505 635594 3285499 4876128 4233556 1931027 4271636 7830888 229134 6660485 8775447 4028765 5465845 6554068 4593621 3910147 6032246 6527558 1251398 3005981 7082407 6571257 6365470 2507248 7805147 5543387 1939601 2867930 3229697 2670868 9284756 4442285 
86
851689 1329153 362176 7822942 2031020 9769577 8845033 3316005 4526457 8379795 5582566 8221873 4509629 5084819 2511478 6096163 1693993 9886945 8381813 2966354 419202 8552679 9477149 952585 3707679 6973821 9691682 7713237 6389555 638988 5890530 8286176 1486141 8156348 8696332 6747552 4440164 9221189 6885428 2583717 4563050 1820575 1682871 1450470 5400360 6983854 5697714 3060319 830114 1530955 2154349 8138258 5947028 9607177 1433372 9736484 1498077 3626047 2410361 3072702 2008744 8129598 3527095 8932071 5751481 7305011 956509 3035323 8186139 6905185 2915725 1457717 5883341 8184343 6765745 410784 7466283 7806166 9685415 941323 6904908 5302516 829176 4380769 5294829 3204116 
46
2428754 6957481 5940539 8131918 8350740 7923194 8070521 4935823 3095043 8997827 3376414 2243165 2004726 3162853 7056845 4110663 4549951 7343571 9428296 8878420 4301202 686574 9023714 6106003 7590403 8448241 3595882 9340366 1148601 8703626 5524041 4350265 1451429 4452740 7346532 8818711 9649781 9161851 869937 9803502 6357493 3932357 6245445 3051955 9551604 7272981 
36
3604791 2438951 3912168 6643907 7901371 6738877 7217532 4873099 8305802 6560778 2141326 3051445 2549721 7625992 5604111 232454 8592558 1391151 4255924 443895 7323395 8116726 5722356 5705554 3421154 8612175 8185076 3060601 6591248 2918593 4613741 2588118 9657586 337094 8309893 1851419 
36
4825924 4253017 5718713 8662106 9174240 1534974 5334077 547693 138177 2284190 9831813 4076078 9464813 7789792 6221779 6493462 5485847 4645970 7055891 181658 9594068 9043724 1058665 8049937 1602783 7824351 6585151 4464842 5898306 7416088 9061109 7207427 9201405 3186880 8406258 2018452 
67
8664427 2757193 6311340 9464275 2632722 3921568 3056865 2072134 3662269 7365857 5997820 4639191 1062161 6225191 3104901 3501855 8040139 2168010 4735288 928074 926774 6628382 6880735 261988 9050087 8413157 3582247 4987999 6324679 627565 7634553 4990816 4404026 9128512 4531349 492044 2325103 9547841 5988751 5290499 2340430 7255301 6107518 3033490 925011 8726361 5229789 6816304 3366149 9736273 9343370 7817162 424300 5133449 335190 4485006 9240821 6559858 3403762 6151904 7535351 3670159 5206647 3162557 7714825 3120431 9070894 
92
1881196 7498488 5978464 3237038 230390 9483434 549039 6140068 6894054 9147444 7414532 6721427 8274999 9767026 3998998 1577950 9727150 3235834 6531041 5429433 3504360 5219677 9745456 5871236 4995300 284669 1362496 7914792 2780707 7703821 2624886 4322932 5771375 6286183 5888771 1146029 4959869 6530943 6203208 1455694 9963541 8395671 2175350 8698864 6779217 1324812 1868125 5659518 9749181 9657219 1262137 4367338 3953285 6206630 9095112 533145 9464318 2377146 6119255 190242 341393 8473624 3984619 1850889 7943143 1637066 8340459 5376566 9581492 5784773 4287408 9635263 6503237 5446168 8228770 3071371 4967332 8332406 7704919 4867005 5624982 3450399 1344700 9524312 5224690 3805715 7014085 135066 4296850 2600454 9163821 3584390 
72
4770290 6988581 3186220 8299192 351879 8059106 9109694 3574278 1504532 3799125 3597299 4732678 3919844 5838906 2222817 4593378 8307465 4539051 8349229 9802518 8284122 6035012 252019 5313588 9919505 1183926 8727023 5909138 4639969 5957207 5887850 2961003 5539985 6529025 7131217 7648596 6022952 3561263 7448170 9019669 5604692 5724136 2638129 8304732 287651 5881497 8685530 9968349 5562965 3268563 8150297 1572005 9545502 8958678 1367631 5583720 5696493 9056329 3980135 6126099 7868281 591488 9336057 6329429 3388859 7523506 2854794 5835029 9342399 9578858 7721392 1324790 
63
9284263 6727920 8634689 2075881 2160028 3775227 1080075 7550765 118302 5130566 3735709 6755909 2937399 7230362 1610003 6146017 9152443 9265474 3679765 5621865 6159603 2387283 4322266 8194866 9856784 2148643 4516297 3152601 4484879 2403516 8796135 3714263 8664799 8532708 6667061 5239417 9204630 7428405 3380221 663986 9903448 4250741 8990022 1777236 7331879 8695797 7526653 2413755 9830189 7401270 6664742 2339250 5047092 3782030 9611895 1646939 6146737 689447 5113295 6671604 5791214 2888573 7182928 
7
2067324 4823760 4036681 3043656 2766300 8847655 1187610 
81
481973 1478358 3344887 6066262 9579745 1090807 8612963 6763740 1605307 4103924 2914325 9223032 5425628 4853533 7412278 4976417 905195 8833235 1293749 1396245 5831500 8924521 6520207 9499511 5407395 5724472 9288563 6037306 6474562 1139626 4300542 7553921 2398634 4529875 2897528 7099291 3505159 1126769 8361024 5646842 8450494 1424699 3082108 3943256 1790209 3238912 5658722 8967223 2116259 6340869 2939933 5461897 4275416 451002 4633468 9389761 9888938 9886520 4297962 3233756 2407435 907243 7212563 7337156 7666860 2159391 837471 503715 2643098 9073719 2751840 7771735 1744599 1066677 6565642 5397786 7807191 2177998 7251950 7806829 8468508 
78
4259869 7875057 5810542 7204218 8329295 755678 6307973 846713 1896427 633896 428834 4209790 8733433 6769993 321522 7997802 9402876 3406555 351457 9037946 7223181 2573310 9268100 978278 5665281 3888644 8803495 7546245 4469154 4015329 1428849 7362570 5152877 8491534 1238900 5030039 2178598 1602064 791675 1115493 2043110 8479650 2912793 8022638 3338973 966481 9092841 1690024 9212227 571652 6916641 2205426 3232537 2414076 303232 5491272 9341270 1110850 5149305 6994593 2658169 9638488 2582261 4821072 2167212 9437443 6914833 7288968 6039424 9016769 4336767 2794129 9949031 1645859 2567693 9950801 6587396 1766061 
90
5883354 3917905 9808821 683128 1169100 4590465 6497941 2744666 5003518 8754780 5931611 9992320 7290337 9838091 8161004 2644507 9525780 6509479 6988341 1702382 2457777 196020 7219915 3931545 5891788 2731941 1606202 4321163 7249212 479195 4728420 1379087 7015880 1410851 4127193 9404522 7314900 9192137 3761698 9708733 1589662 6317640 5962885 8397438 4875289 1927284 6304865 8768810 2097537 7520536 4011144 936270 9624781 8133379 4383412 4751930 6842042 2763675 4924153 8422970 4766113 4353377 3109069 1892681 5974646 9786616 7183971 1100320 4748141 6747642 5575650 8169231 4776517 5958197 6442033 635954 8534773 626364 2756714 318355 396897 6942185 143543 3169860 2909915 3654750 1315092 938801 843560 2041851 
29
317737 4404848 2764135 6429693 4008902 4756369 6396300 492410 5783907 313441 1051515 2683785 618845 4327689 6882603 7111814 2926128 9554520 5783287 8620587 6721907 4463664 7277942 5423588 956638 4585582 2103517 7638358 3601722 
71
5766377 6915619 256907 9920196 3093523 7119189 2680446 2264632 8436478 9927713 5559135 2657258 799673 589158 5534919 2847637 3108786 2692282 8106704 1484665 1071656 8987809 5480594 8831660 1033972 577811 5770215 5061085 3633892 6185032 7902948 3709103 7187164 9276037 1160858 8312014 1710751 9507662 9585734 4576694 7922020 3259378 2837847 9702149 8023950 9553617 4947123 4344962 903670 9595928 6100837 3100975 6962005 4589584 9649325 6381848 2533780 3326388 7429562 3243107 9578926 3911647 6876521 5809617 8003984 7701904 976037 6702644 8161390 496981 3475817 
27
435858 8072790 7654205 9174595 9649672 3542435 3748739 4885034 719022 7910482 397046 8044742 361510 9530220 4057865 592826 9941068 3371680 1322643 3497186 6507123 4059324 3725567 3404512 4372772 9887462 2225161 
39
1078306 5349582 7908283 5125578 943386 5246675 9829161 6301746 7715157 827417 4692544 5994457 3252363 7089795 8124897 1083848 4215347 6479906 8869751 1616123 2657944 971150 8506002 3198712 3298338 8141563 5335879 6517014 7058105 6121347 3186248 5443039 5239602 6551761 5832560 5616381 5634159 2902866 6149020 
8
8172194 1404526 1196695 9724206 9621699 4868007 4206956 3199398 
70
4209555 1716863 9768916 6181920 1448332 3823157 810811 1326788 5797422 4605033 6466179 6054235 3969460 362848 9958172 4924725 984240 7034890 1237430 6461796 3409978 7973760 9192509 1469673 9872638 2639085 9821670 8145350 1625287 961642 1508478 1504837 5173931 4692295 9583887 6878873 6581611 9158261 2978104 7786471 1141874 6986731 8002208 437647 9900875 7897892 8821865 3463293 8478038 1711249 8218734 5182910 500215 5309614 2726979 4942622 2195859 6971159 3947232 3222912 7511370 8559042 2012644 4498148 2796647 8630864 4163091 3874878 6989583 6703414 
76
455105 9664117 1469599 2980591 3271153 7816617 4694784 4271159 5447993 909618 4754904 4299806 4280031 7792751 4682135 2111626 9730572 2524356 9909124 5567910 4690569 559863 5008481 1740087 3197780 9211030 7507817 769775 7604263 1281020 8999743 6162555 2146291 7780475 1543501 5417884 5428787 241991 4641191 7652506 7367061 6851214 6676167 887767 4883540 4896882 8883892 2117808 8631154 8111784 7995036 7136920 511667 7375375 317359 106028 5455219 8050414 2372640 9839765 7979904 6695214 4750158 2010870 8540164 7489281 2084930 2727719 7090405 7771959 461219 832144 6188358 3050349 6853051 3819214 
46
4193511 7710855 2323625 5927949 3584975 5314607 2837314 2480841 9164204 1747369 2039461 5917295 7580041 2155651 3286128 7064897 9916505 7521207 4567556 1678603 5693412 2267129 4758744 8428871 4142132 9707549 2780879 296118 3104925 9728637 1156256 6236925 8686300 5551056 9518735 972448 9722591 7005542 1968427 7405118 8649727 4858515 9510937 1994127 8618477 5203816 
57
7105686 3224083 4332359 1818783 4910520 8413113 1994485 6627831 1287312 9127075 5365044 1934579 2316714 2145041 4466759 935556 9514857 8684535 1307369 5860268 174999 964979 5477111 2910155 4958535 625729 4130040 1016781 5592008 7319173 4344829 5674539 4213169 6716377 9577400 8533013 6497317 3983206 2561211 8338865 956202 877378 1026627 1450691 7550367 8532632 9258607 4766834 7465941 5920968 4706388 8155611 9628722 3743898 8537619 3061651 8564782 
32
4695433 3379850 5800004 5717071 2546483 1110098 4395825 2092549 6540856 478871 8794634 8649149 2776581 2002164 7798208 8755083 9576223 6677268 2966274 6870447 9757204 5360252 9353612 6428819 3595626 7221543 9935523 2902250 3252310 8043050 6689604 3607752 
61
3748582 4100932 8956587 4942567 1691954 6711292 8389620 1585414 8456932 9736674 2293627 1796940 2947783 9259257 6069465 4158381 582719 2105967 5510719 6301738 623569 8259309 4326180 3008989 1638316 7091513 3865651 1940262 649468 7258754 7082143 3282469 7559750 6447117 5386453 4485104 5507524 4633031 7861253 3846070 3221567 4166413 2474410 5296130 8594372 1405409 9935136 6644684 276993 6216886 3498556 2548527 7963331 2433694 8931539 8507330 4259696 390116 1969637 6804329 3536854 
37
1724921 5098588 9361818 3811678 4672247 7320311 3638610 181086 2980444 7335285 1296911 2558374 7618442 9900042 641318 9044809 6722252 4565777 8104768 8436817 497093 9635748 3608679 2011718 1292073 4267660 945480 7140504 533631 8328302 1154997 2162589 593424 4902595 9251582 5339688 6068785 
97
1142113 1712819 3475590 9163053 2810621 1506104 2663115 1105138 8483472 708154 9189586 8651397 5752293 9474250 4066789 9848311 2393416 8778214 9471577 1114393 2710346 8801464 3088617 3366100 7319665 5547462 5128932 9900724 5757827 7083663 2999864 8936817 4785767 3880845 1654983 1518020 6056143 1533791 6684363 296951 7359832 6582935 3383705 5382155 3968933 9801109 1530983 6750412 5029607 9563839 3945095 2838473 835264 6768489 6470523 3384707 895218 9024439 9906059 5339657 4980357 7197609 3375117 1671746 4202378 6467855 3422319 2961533 8010123 3350032 3041757 1873170 6367155 1845168 6430139 5492910 8978639 9744219 6767114 2289576 6724175 8665685 468309 8021118 5916639 3123182 5873671 2489598 4325695 950873 3218396 2890843 9955684 198815 8179749 2510425 8847146 
82
3660938 8750044 4700604 1704108 1858767 3309349 1064117 7578257 2857843 963041 5723716 2878053 3794869 8559580 5374097 7879346 6026704 9618299 4033056 5083804 1800359 640286 642259 1551617 4309629 2388880 8487285 5376150 364641 223193 8046493 8849636 6387142 6125104 4370297 8095118 2433335 6128389 4986813 205492 4824463 4816375 522020 1637123 2064819 8477797 8072642 7057573 8266245 4513441 5357739 7422264 2412113 7105185 8779473 1659969 434666 4926722 1125438 4147876 482253 9254763 8233674 6186755 9390278 9805118 378623 2021670 2709795 1581009 1471633 4373917 5111045 9275908 170322 4846354 8864407 2994145 349845 6504706 9474617 9184236 
93
4645280 131421 1274243 7792324 2516103 2251507 3982955 9644843 8190991 339992 8253208 5832638 6988464 7086795 3257826 6184102 7233303 3377172 1842133 3767526 3720406 4781490 4272738 3050366 8506018 410364 2591058 6194335 489020 583669 5218810 7269842 6328424 6185779 3118587 9231467 9914945 267558 9751256 6706171 2294358 8771228 4182732 6070438 6630355 6253292 8000943 1417371 2401768 9731522 1808608 4877013 8375736 5998044 4599345 7594106 4208538 3878180 7895742 6984178 9422517 4894816 2806655 6063966 8395285 2999481 8911325 6386461 9860833 5783654 5573010 9843776 1297677 433236 5925576 5972270 2126261 1861865 1888618 5442715 5121243 5512157 4475469 6349727 6100874 7807614 9129057 1225787 4420780 8253018 4787695 6156223 5325028 
67
2949011 6504563 7308025 6783345 4593064 4902123 9316662 1892987 1848453 3508149 8405892 7003867 3011992 6921829 7327005 2529275 5288076 955378 5404250 9066362 7861615 6093216 4126484 7769020 964098 3095257 8544032 447160 9650103 4101828 482949 828209 4296730 6917817 668484 2397921 3278894 3738005 5137307 6682066 5163241 6749097 4680680 233292 3842177 4938437 6720066 7355943 929017 2433796 376417 765008 9842242 2113219 6305844 529954 3505483 9934107 8604864 4130653 8812130 1977529 6035902 8700899 1245512 7289157 2459003 
22
5350879 3068927 7958962 5329150 7375835 4991590 2378259 3974565 2185834 2553535 7401667 5775387 4978231 3023112 3729685 6242484 7275457 2217912 946013 5728891 2877270 9586010 
83
6130494 9942510 5489067 843395 2547823 7448606 8665131 4596961 8059395 7697884 5507775 6872197 8457549 3147009 8813606 739299 7230045 2663032 8220822 1318132 5495095 571681 2792488 828418 7130739 8850015 7705669 2283683 5854762 4822252 8139171 7776335 3438195 5866338 7297784 2102778 3582736 1503730 1404831 1309836 2100766 6687100 2568948 9372619 9681298 7720721 6500667 2491652 6474407 8329655 6748250 3425631 2789887 9089216 5297939 9309631 5807299 9645323 6031821 6307550 5940932 8690079 4308026 6048447 6983409 3318459 8717321 4714310 5646796 8949227 6694490 9980820 9075278 9429203 4738065 7334308 2790808 5920258 7474195 7621847 237564 629632 9488741 
16
2625702 8722140 4187015 6364915 3551255 4286855 5897704 3645121 2415098 9111996 6390685 2558278 5899130 7328070 4255705 791336 
38
287105 5469687 8268245 572858 5735384 5786375 3762686 9478415 9063133 9643742 1209614 759178 2466738 6347428 3977969 2917462 769152 3811885 8098668 5678974 1928179 6935237 7831238 2763026 5084839 7200733 7609425 9078217 540824 517777 3525727 4063333 6355577 6227362 3199766 7821773 9795691 3296325 
16
3471067 4116751 9067108 7158639 7654362 7241841 2518756 4265984 9475172 8651159 7647171 4477242 7025637 1091516 5337633 6142512 
6
5051286 8796739 6609382 3837218 4235791 3688327 
70
1755831 7236677 3693104 5772450 735029 2135139 7395068 2556961 2234405 6755916 8449153 3288941 6333421 6708243 2385423 5724189 9675825 8764731 2075608 5694773 7434026 4704087 4963909 3292760 1232645 196521 4316157 7457919 9589764 491183 1833424 683530 2963445 3374704 6286730 9309939 4923224 3214493 9416193 1812425 3281348 924764 7104656 5871883 462214 6263478 635103 2412749 2735839 2955631 7489198 8653295 1367412 9232666 6331641 3309382 7717856 7138408 1904652 6145181 6640856 2563322 2559732 3366127 2922338 3080257 5779702 6586375 8963901 9572416 
9
4424093 5342093 6179119 7507312 4068103 7003578 1414141 8410252 9428773 
51
6271979 8744634 6771035 3151076 5058352 8059521 8959920 1345889 5034696 1522467 6846372 7219573 7646835 3353890 5521719 7492201 7222640 4594435 3650213 1882453 6367311 3932363 8240682 4201364 4932832 2051090 6261125 717201 1687569 9426918 1040749 4502869 7953207 8739944 3367095 7674220 5619072 9396405 7282922 7782592 6224010 1644074 8601592 5512441 3147311 2136602 1086320 1462144 6837006 8223198 6784823 
93
1587583 1108365 3419916 8292886 6199188 6464841 6179486 3034266 9327479 7838394 7835631 6467597 4692088 7425024 729101 8209764 1902442 4617788 9181745 2358044 7184532 2936300 5320898 7072197 3474571 1138543 1640595 2655712 241193 4412431 1884258 8569275 6843147 4197721 4921637 6032842 8804006 4586534 6488717 4287384 2267311 7531799 9758251 8966934 3673717 5163076 7361708 4018361 8791587 2448766 5475301 8976208 4546495 1668822 1104091 3101938 1259160 9965607 7559950 7022392 8763799 4701735 5794706 3132811 349171 8053320 398387 9611358 5642504 1694776 1007193 9602600 2078227 792668 5054587 1884579 5815873 9263988 8032115 811061 8517353 4681650 8641905 7926819 198605 4402106 5747337 7916843 1130049 4966490 5935636 7737028 7802865 
31
9510511 4875938 1409120 5253219 7713330 8127634 3471262 191354 6813632 2279233 9913411 3348016 7385110 5590104 8723330 9749559 2113568 3803711 8280362 9927215 5941494 1534963 9079076 3776661 5388830 885419 3080220 4215090 9706006 1741785 2172970 
23
238605 8504937 366590 1776514 5469879 4721387 545729 8934878 5578924 8002712 2601807 625562 5496439 5572494 1635636 3782409 6455564 6387128 1784616 5282893 4094894 3187823 4052355 
94
8163016 8381805 1038576 3196470 1984941 4370183 5802791 4463719 9451226 1997479 346778 2550947 4935169 4922249 4466921 5547492 7665105 856256 6023253 9953301 2656074 2887863 6077267 2395479 3158442 6935855 1517181 4451679 8541477 5797162 1656353 1154662 1635218 6384593 3991312 383051 5287496 5603495 4063197 1600985 2754927 4533147 4196365 6043602 9335011 4980968 1717629 4650893 5680667 6267300 2073166 3951161 7081006 8684359 5560695 9952733 2363339 8005069 8927802 6867694 129975 9875458 4750811 4042454 9869611 2621260 5059351 8620866 1645214 7953288 2075322 5679148 6382169 4842879 9692049 5275029 1335501 6164906 935401 3281746 6928321 7273101 4574635 7832685 201927 3382626 4807899 3561747 751012 5027172 4750422 3130217 4879951 9354582 
