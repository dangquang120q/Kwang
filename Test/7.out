28591635
29009667
29693826
28417703
28424768
28792556
25445795
29798831
27956891
29574314
28863641
28102974
29374517
29393006
29004887
26862319
28890694
28748389
29479748
29690372
29590421
17708096
29355203
29538320
29639232
25813465
29550058
29478202
27204914
27518099
29731685
29479461
29367733
28720979
29268950
28931067
28897608
29762467
28898034
29619554
29426452
24946639
29604628
25162206
28917848
27193439
20457407
28838005
25346337
27783243
29335216
29590185
25442527
29781492
